prog/weaveutil/addrs.go

```diff
@@ -27,10 +27,25 @@ func containerAddrs(args []string) error {
 		return err
 	}
 
+	containerArgs := []string{}
+	for _, cid := range args[1:] {
+		if cid == "weave:allids" { // expand to all container IDs
```

Comment:
Nitpick, but I'd probably implement `weave:allids` as a flag, i.e. `weaveutil container-addrs --all`. It could also cover `weave:expose`.